services/agents/log_level.go

```diff
@@ -0,0 +1,17 @@
+package agents
+
+import (
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/version"
+)
+
+// Log level available in exporters with pmm 2.28
+var exporterLogLevelCommandVersion = version.MustParse("2.28.0")
```

Comment:
I guess it will go to 2.29

Reply:
So main dependency is support in exporters side. This support added in 2.28.0 https://jira.percona.com/browse/PMM-9840